SPECS/tdnf/tdnf.spec

```diff
@@ -47,14 +55,14 @@ BuildRequires:  glib
 BuildRequires:  libxml2
 BuildRequires:  python3-pip
 BuildRequires:  python3-requests
-BuildRequires:  python3-setuptools
 BuildRequires:  python3-xml
 %endif
 
 %description
 tdnf is a yum/dnf equivalent which uses libsolv and libcurl
 
 %define _tdnfpluginsdir %{_libdir}/tdnf-plugins
+%define _tdnf_history_db_dir %{_libdir}/sysimage/tdnf
```

Comment:
Since we're here, could we move these macros to the top of the spec? It makes it easier to find them.

Reply:
Sure thing!